AGENTS.md

```
@@ -0,0 +1,87 @@
+# Agent Instructions
+
+This file provides guidance to Claude Code and OpenCode/Codex when working with code in this repository.
+
+## Overview
+
+This is Vatsal Sanjay's personal website built with Jekyll and hosted on GitHub Pages at https://comphy-lab.org/vatsalsy. The site serves as a redirect hub to the CoMPhy Lab website (https://comphy-lab.org) for research content, team information, and contact details.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)